ee-feature-pack/galleon-shared/src/main/resources/feature_groups/servlet-host-secondary.xml

```diff
@@ -52,6 +52,7 @@
             <feature spec="host.core-service.discovery-options.static-discovery">
                 <param name="static-discovery" value="primary"/>
                 <param name="protocol" value="${jboss.domain.primary.protocol:remote+http}"/>
+                <param name="host" value="${jboss.domain.primary.host:localhost}"/>
```

Comment:
@ehsavoie this seems wrong. Should not this be replaced with `<param name="host" value="${jboss.domain.primary.address}"/>` instead?